### ISSUE TITLE
Spelling in Operations Weekly Report

### ISSUE THREAD

**Issue by brotzmanmj** [Analyticsphere/metricsReportsRequests#192]
Hi @KELSEYDOWLING7 I happened to see a spelling error in the title of Table 1.4 in the operations weekly report from Dec 9. Can you have someone go through the report carefully and find any other errors so you can correct them?

Thanks

**Comment by KELSEYDOWLING7** [comment]
Leila, would you have time to do this for me? The ops report Box location is: https://nih.app.box.com/file/1720713318593

**Comment by lorszag** [comment]
@KELSEYDOWLING7 Yes! I should have time either tomorrow or friday. Does that work?

**Comment by KELSEYDOWLING7** [comment]
Yup, thank you!

**Comment by lorszag** [comment]
I went through once and didn't find any other spelling errors (besides the one Michelle already mentioned) but found some minor grammar/capitalization things. Listed below by what table/figure they came in. I'll go through this again with fresh eyes tomorrow to see if anything else sticks out. @KELSEYDOWLING7 

- Figure 1.1: Graph title doesn’t make sense. Maybe “Number of Participants Verified & Response Rate Over Time, All Sites"
- Figure 1.2: graph title doesn’t make sense? Maybe “Number of Participants Verified Over Time"
- Figure 1.3-1.13: graph title —grammar seems off with where the date is placed, currently Weekly Number of Invitations and Study Activity Completion 2024−12−08 All Sites. Maybe "Weekly Number of Invitations and Study Activity Completion for All Sites Through 2024-12-08"? 
- Table 1.4: period is in the wrong place write after the table number and before the table title; Participants is spelled wrong in “verified participants"
- Table 1.5: Capitalization of In in Signed-In inconsistent  (Also applies to table 1.6)
- Table 1.7-1.8: capitalization of Yet in “Not Yet Verified”
- Table 1.10: Capitalize Be in “Cannot Be Verified”
- Table 1.14: should be Phone Call Invitation instead of Phone call invitation
- Table 1.15: Table 1.16 consistency of “Aging out of study” capitalization
- Table 2.4: remove comma from title
- Table 5.5: the note has something weird in it make cell[1]
- Table 6.1-6.2: titles don't make sense?
       - Note for table 6.2 Refusal is incorrectly capitalized
- Table 7.2: note says “table was ran” should say “table was run”, second note says “are assumed English”I think it should be something like “are assumed to have English as their preferred language”
- Table 7.3: remove comma from title

**Comment by lorszag** [comment]
Okay only found one more thing: 

- Table 4.6-4.16: Extra space between "Not" and "All" in row labels

**Comment by KELSEYDOWLING7** [comment]
Thanks! I updated all of the capitalization and grammar issues. The changing of titles I didn't do, as Operations has named all titles. @brotzmanmj Do you agree with any of the title suggestions above?

**Comment by KELSEYDOWLING7** [comment]
@brotzmanmj  @robertsamm  Following up on my question above

**Comment by robertsamm** [comment]
Thanks @KELSEYDOWLING7, I'll review the titles this morning and get back to you!

**Comment by robertsamm** [comment]
Hi @KELSEYDOWLING7 I reviewed all the proposed title changes and they look reasonable to me. Please make the updates. Thanks @lorszag for reviewing!

For Table 6.1-6.2: titles don't make sense? - could change to 'Number of Refusals and Withdrawals Among All Verified Participants That Requested It' and 'Reasons for Refusal or Withdrawal Among Verified Participants That Requested It'

**Comment by KELSEYDOWLING7** [comment]
All set! This will be applied starting this Monday
